R/plotDataSetList.R

```diff
@@ -894,9 +907,12 @@ Plot.FV.Histogram.DataSetList <- function(dsList, runtime, plot_mode='overlay'){
     # skip if all target samples are NA
     if (sum(!is.na(fce$'f(x)')) < 2)
       next
-
-    res <- hist(fce$'f(x)', breaks = nclass.FD, plot = F)
+    
+    if (use.equal.bins) breaks <- res1$breaks
+    else breaks <- nclass.FD
+    res <- hist(fce$'f(x)', breaks = breaks, plot = F)
```

Comment:
I prefer `f(x)`... please don't bother

Reply:
If we change it that way, we need to add f(x) as a global variable since otherwise the CRAN checks give a note, so I think it is better to leave it as is.